internal/operators/cnv/manifest.go

```diff
@@ -140,3 +144,34 @@ metadata:
   namespace: "{{.OPERATOR_NAMESPACE}}"
 spec:
   BareMetalPlatform: true`
+
+const cnvHPPManifest = `apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+    spec:
+      imagePullPolicy: IfNotPresent
+      storagePools:
+        - name: sno
+    pvcTemplate:
+      storageClassName: localblock-sc
```

Comment:
I have seen this storage class name referenced throughout the repo but couldn't find whats in charge of setting it
Example occurrence:
https://github.com/openshift/assisted-service/blob/master/internal/operators/lso/manifest.go#L71

Reply:
This will be available only if LSO is installed.
Is this a requirement for CNV+SNO?

Reply:
Yes, we want this to be a requirement for CNV+SNO
In what flows is LSO disabled?

We could also use other storage options but I don't think we have a better option than LSO
which can already be installed here

Reply:
Sorry, LSO is a requirement for CNV:
https://github.com/openshift/assisted-service/blob/master/internal/operators/cnv/cnv_operator.go#L59